docs/source/user-guide/getting-started.rst

```diff
@@ -7,65 +7,98 @@ Project for the first time.
 
-     anaconda-project add-command plot .
+Create an example notebook-based Panel app
+==========================================
 
-   NOTE: By default, Bokeh looks for the file ``main.py``, so you
-   do not need to include this in the command string after the
-   "plot" command name.
+In this section, we will create a new notebook called
+``Interactive.ipynb`` via **either** of the following methods:
 
-#. When prompted, type ``B`` for Bokeh app::
+#. Download this `quickstart`_ example (either right-click and "save-as"
+   into your project directory or use ``curl`` on a unix-like platform)::
 
-     Is `plot` a (B)okeh app, (N)otebook, or (C)ommand line? B
-     Added a command 'plot' to the project.
-     Run it with `anaconda-project run plot`.
+    $ curl https://raw.githubusercontent.com/Anaconda-Platform/anaconda-project/master/examples/quickstart/Interactive.ipynb -o Interactive.ipynb
 
-#. Run your new project::
+   Note that this example is taken from a larger, more full-featured
+   `hvPlot interactive`_ one that will work as well, if you would prefer
+   to download that.
+
+#. Alternatively, you may launch a Jupyter notebook session with::
+
+    $ anaconda-project run jupyter notebook
+
+   and create a notebook called ``Interactive.ipynb`` with the following contents placed into a cell:
+
+.. code-block:: python
+
+    import xarray as xr
+    import hvplot.xarray
+    import hvplot.pandas
+    import panel as pn
+    import panel.widgets as pnw
+
+    ds = xr.tutorial.load_dataset('air_temperature')
+    diff = ds.air.interactive.sel(time=pnw.DiscreteSlider) - ds.air.mean('time')
+    kind = pnw.Select(options=['contourf', 'contour', 'image'], value='image')
+    interactive = diff.hvplot(cmap='RdBu_r', clim=(-20, 20), kind=kind)
+
+    pn.Column(
+        pn.Row(
+            pn.panel("https://hvplot.holoviz.org/assets/hvplot-wm.png", width=100),
+            pn.Spacer(width=20),
+            pn.Column(
+                pn.panel("## Select a time and type of plot", width=400),
+                interactive.widgets()
+            ),
+            pn.panel("https://panel.holoviz.org/_static/logo_stacked.png", width=100)
+        ),
+        interactive.panel()
+    ).servable()
 
-     anaconda-project run
 
-   NOTE: If your project included more than one command, you
-   would need to specify which command to run. For more
-   information, see :doc:`tasks/run-project`.
+.. _hvPlot interactive: https://raw.githubusercontent.com/holoviz/hvplot/master/examples/user_guide/Interactive.ipynb
+.. _quickstart: https://raw.githubusercontent.com/Anaconda-Platform/anaconda-project/master/examples/quickstart/Interactive.ipynb
 
-   A browser window opens, displaying the clustering app.
+Run your project
+================
 
+#. Register a new command to launch the notebook as a `Panel`_ app::
+
+     $ anaconda-project add-command --type unix dashboard "panel serve Interactive.ipynb"
```

Comment:
Perhaps offer the windows variant as well and say `--type unix` is also used on Mac computers.

```
anaconda-project add-command --type windows dashboard "panel serve Interactive.ipynb"
```

Reply:
Here's a good example of where we'd like not to have to specify the platform! 

Also, without https-options, I think this will only work locally and not inside a container...

Reply:
Good call on platforms. I'll add a note for windows users and totally agree. I was surprised to be asked about what type of dashboard when I didn't include `--type unix`. That's the type of implicit behavior we should remove.

Regarding Docker, I sort of expected this would only work for local. I can put a comment about that. Would personally prefer just a note instead of introducing more concepts in quickstart.

Reply:
Yes, dockerize would require `supports_http_options`